tracer/src/Datadog.Trace/TraceContext.cs

```diff
@@ -95,8 +95,6 @@ public Span RootSpan
         /// </summary>
         internal IastRequestContext IastRequestContext => _iastRequestContext;
 
-        private TimeSpan Elapsed => StopwatchHelpers.GetElapsed(Stopwatch.GetTimestamp() - _timestamp);
```

Comment:
I like that this cleans up `TraceContext`'s exposed API and moves all the time-related members to `TraceContext.Clock` 👍🏽